docs/first-steps/policy-enforcement.md

```diff
@@ -43,34 +43,117 @@ That said, the below strategy is designed for more real-world scenarios where po
+
+|Policy Mapping Setting |   |
+|--|--|
+| Policy Mapping Name |`cluster-wide-enforcement`|
+| Linked App Group| `match-all`|
+| Policies | `polaris.memoryRequestsMissing, polaris.cpuRequestsMissing, ...`|
+|Context| Admission|
+|Block| false|
+
+
+### 3. Configure "Scoped Policies" targeting specific Kubernetes resources
+_Scoped Policies target resources associated with teams, apps, or business units with specific policies (such as security or reliability checks)._
+
+**3A. Create an App Group focused on specific workloads** -- [example here](/features/app-groups#matching-a-very-strict-workload-by-every-available-option)
+
+|App Group Setting |   |
+|--|--|
+| App Group Name |`payment-business-unit`|
+| Match | `namespace: payment-apps` |
+| Exclude| _nothing_ |
+
+
+**3B. Create a Policy Mapping with specific policies, and link it to that App Group** -- [example here](/features/policy-mappings#scan-only-polaris-livenessprobemissing-policy) 
+
+|Policy Mapping Setting |   |
+|--|--|
+| Policy Mapping Name |`scoped-policy-enforcement`|
+| Linked App Group| `payment-business-unit`|
+| Policies | `polaris.runAsRootAllowed, polaris.insecureCapabilities ...`|
+|Context| Admission|
+|Block| false|
+
+
+### Important: Policy Mappings are additive
+
+A single Kubernetes resource can be subject to multiple Policy Mappings. The `policies` within a Policy Mapping are additive, so that Kubernetes resource will be evaluated against the unique, aggregate list of policies across all those Policy Mappings. For example:
+  1. `Policy-Mapping-A` has 2 policies `polaris.runAsRootAllowed`, `polaris.cpuRequestsRequired`
+  2. `Policy-Mapping-B` has 2 policies `polaris.memoryRequestsRequired`, `polaris.cpuRequestsRequired`
+  3. Workload `api-server` matches both Policy Mappings, and therefore will be scanned for 3 unique policies:
+      - `polaris.runAsRootAllowed`
+      - `polaris.cpuRequestsRequired`
+      - `polaris.memoryRequestsRequired`
+
+> NOTE: [It's possible for Kubernetes resources to be subject to conflicting `block` directives](/features/policy-mappings#when-a-kubernetes-resource-has-conflicting-block-directives)
 
 ## Implementing Stage 1: Awareness
 
-_In this stage, we want to warn users when their YAML code or Admission requests contain misconfigurations found in our Cluster-wide Policies or Scoped Policies._
+_In this stage, we want to setup Insights to report in-cluster policy violations._
+
+_We also want to report (but NOT block) Admission requests and CI scans that contain policy violations. This is to help with raising awareness of configuration standards._
+
 
-_The simplest way to do this is enabling Admission Controller in your cluster and setting it to Passive Mode (which is enabled by default). Additionally, you can install Insights in your CI pipeline to surface misconfigurations sooner in the process._
 
-### 1. Install Admission Controller
+### 2. Install Admission Controller
```

Comment:
Why did this numbering change? I don't see a `1`

Reply:
Good point, fixing now